tools/tidb-binlog-cluster.md

```diff
@@ -0,0 +1,566 @@
+---
+title: TiDB-Binlog cluster 版本用户文档
+category: advanced
+---
+
+> cluster 版 TiDB-Binlog 尚未发布正式版，本文档用于测试环境部署。
```

Comment:
After the cluster version of TiDB-Binlog is formally published, do we need to remove this document? https://github.com/pingcap/docs-cn/blob/master/tools/tidb-binlog.md

Reply:
yes, we can keep `tidb-binlog.md` this file, only replace the link in home page.

Reply:
@WangXiangUSTC Got it.